index.js

```diff
@@ -0,0 +1,28 @@
+import Books from './modules/Books.js';
+
+import { DateTime } from './modules/date.js';
+
+import {
+  even,
+  showhide,
+} from './modules/elements.js';
+
+Books.displayall();
+
+document.querySelector('.add').addEventListener('click', even);
+
+const timeDate = document.querySelector('.date');
+
+const dateAndTime = () => {
+  setInterval(() => {
+    const date = (`${DateTime.now().toLocaleString(DateTime.DATETIME_MED)}`);
+    timeDate.innerHTML = date;
+  }, 0);
+};
```

Comment:
- [ ] Please make sure the current date is displayed below the navigation bar using luxon library

<img width="941" alt="Screenshot 2023-06-10 at 4 51 46 PM" src="https://github.com/vikas924/awesomebookes6/assets/44798044/111cb81c-e655-4357-90db-f6d9f7b76c31">